templates/full/alarm.vm

```diff
@@ -1,9 +1,16 @@
 #set($subject = "$device.name: alarm!")
+#{if} ($position.getString("alarm") == "lowBattery")
+#set($batteryPercentage = $position.getString('batteryLevel'))
+#set($batteryVolts = $position.getString('battery'))
+#set($batteryString = " (#{if}(${batteryPercentage})${batteryPercentage}% #{else}#${end}#{if}(${batteryVolts})${batteryVolts}V#{else}#${end})")
```

Comment:
Can we remove trailing space?

Reply:
I added space between `lowBattery` and the opening bracket on purpose so that it looks like this:
`DeviceA alarm: lowBattery (11.2V) at 2022-01-29 06:05:24`
instead of this
`DeviceA alarm: lowBattery(11.2V) at 2022-01-29 06:05:24`

Reply:
I'm talking about trailing space after the %.

Reply:
Done.